policy/policy.go

```diff
@@ -164,6 +168,9 @@ type RuleSet struct {
 //
 // The AND match here must be able to say: ip == foo and tcp port = bar
 func (rs *RuleSet) MatchRule(r *Rule) (bool, bool) {
+	if *r == ignoreRule {
```

Comment:
how bad does it become if we create an IGNORE operator and use it on the ruleset instead?

Reply:
The problem is that at the ruleset level we will have two sides (left and right) but we might only want to blank one of the rules. I think it's better at rule level. Would it be more clear if instead of using a marker object to designate ignored rules we added an `active` flag (private, not serialized) to the rule struct?